CMakeLists.txt

```diff
@@ -358,6 +367,22 @@ target_include_directories(_pythia8 PUBLIC
   ${cpp_dir}
 )
 target_link_libraries(_pythia8 PRIVATE libpythia8)
+
+# Add . directory to rpath for MacOS and Linux
+# It enables _pythia8 to search for dependencies (libpythia8)
+# in its own directory. Windows does it by default.
+set(pythia8_rpath_fix)
+if(UNIX AND NOT APPLE)
+  set(pythia8_rpath_fix "$ORIGIN")
+elseif(APPLE)
+  set(pythia8_rpath_fix "@loader_path")
+endif()  
```

Comment:
This condition can be written more succinctly by reversing the conditions. If APPLE is a subset of UNIX, then do
```
if (APPLE) ...
elseif(UNIX) ...
endif()
```

Reply:
Done